src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -605,7 +636,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   }
 
   def workflowGcsPath(descriptor: WorkflowDescriptor): String = {
-    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, jesConf.executionBucket)
+    val bucket = descriptor.workflowOptions.getOrElse(GcsRootOptionKey, jesConf.executionBucket).stripSuffix("/")
```

Comment:
This makes sense, but can you describe (here, no need for comment) what issue led to this?